### PR TITLE
graphEditorContainer panTo with javafx.geometry.Pos added

### PR DESCRIPTION
**Pull request by danielemaddaluno**
I have added this `graphEditorContainer.panTo` feature which I have found on the graph-editor from tesis-dynaware.

### CODE REVIEW

**Comment by eckig** [comment]
Thanks for the idead and implementation.

1. Please use blanks (4x) instead of tabs for indent.
2. Please put curly braces `{` on a new line.
3. Instead of creating a new `enum` you could reuse some from JavaFX like `HPos` and `VPos`?

File: demo/src/main/java/de/tesis/dynaware/grapheditor/demo/GraphEditorDemoController.java [hunk 1]
```diff
@@ -144,8 +145,22 @@ public void initialize() {
...
+     * return non-zero values.
+     * </p>
+     */
+    public void panToCenter()
+    {
+        graphEditorContainer.panTo(Pos.CENTER);
+    }
 
-    @FXML
+	@FXML
```
Comment (eckig) [inline demo/src/main/java/de/tesis/dynaware/grapheditor/demo/GraphEditorDemoController.java:163]: Tabs vs. spaces...  :-)

Comment (danielemaddaluno) [reply demo/src/main/java/de/tesis/dynaware/grapheditor/demo/GraphEditorDemoController.java]: Right, sorry 👍

**Review by eckig — DISMISSED** [review]

File: api/src/main/java/de/tesis/dynaware/grapheditor/window/PanningWindow.java [hunk 2]
```diff
@@ -10,8 +10,10 @@
 import javafx.collections.ObservableList;
 import javafx.event.Event;
 import javafx.event.EventHandler;
+import javafx.geometry.HPos;
```
Comment (eckig) [inline api/src/main/java/de/tesis/dynaware/grapheditor/window/PanningWindow.java:13]: Is that import really needed? I only see usages on `Pos`?

Comment (danielemaddaluno) [reply api/src/main/java/de/tesis/dynaware/grapheditor/window/PanningWindow.java]: Yeah, we can remove it. Thank you again.

**Review by eckig — APPROVED** [review]

File: api/src/main/java/de/tesis/dynaware/grapheditor/window/PanningWindow.java (hunk 2 above)
Comment (eckig) [reply api/src/main/java/de/tesis/dynaware/grapheditor/window/PanningWindow.java]: Thanks for your work!

**Pull request merged**